gef-ui/server/controllers/amendments/helpers/create-amendment-reference-number.helper.test.ts

```diff
@@ -64,10 +69,23 @@ describe('createReferenceNumber', () => {
     await expect(createReferenceNumber(dealId, facilityId, userToken)).rejects.toThrow('Submitted amendment was not found for the deal');
   });
 
+  it('should throw an error if getFacility details object is null', async () => {
+    getFacilityMock.mockResolvedValue({ details: null });
+
+    await expect(createReferenceNumber(dealId, facilityId, userToken)).rejects.toThrow('Facility was not found');
```

Comment:
```suggestion
const response = await createReferenceNumber(dealId, facilityId, userToken);
expect(response).rejects.toThrow('Facility was not found');
```

Can you please write these in `Arrange-Act-Assert` pattern.

Reply:
done